packages/e2e-tests/scripts/buildkite_run_fe_tests.ts

```diff
@@ -234,32 +234,60 @@ export default async function run_fe_tests(
   // Debug replay:cli by default.
   process.env.DEBUG = "replay:cli";
 
-  let webProc;
+  let dashboardProc;
+  let devtoolsProc;
   if (runInCI) {
     // Get ready.
     execSync("cp .env.sample .env", {
       stdio: "inherit",
     });
 
-    // Start the webserver.
-    webProc = exec("npx yarn dev", (error, stdout, stderr) => {
-      if (error) {
-        console.error(`yarn dev exec ERROR: ${error}`);
+    // Start the dashboard server
+    dashboardProc = exec(
+      "pnpm dev -- -p 8080",
+      {
+        env: {
+          ...process.env,
+          DEVTOOLS_URL: "http://localhost:8081",
+        },
+      },
+      (error, stdout, stderr) => {
+        if (error) {
+          console.error(`dashboard server ERROR: ${error}`);
+        }
+        console.error(`dashboard server stdout: ${stdout}`);
+        console.error(`dashboard server stderr: ${stderr}`);
       }
-      console.error(`yarn dev stdout: ${stdout}`);
-      console.error(`yarn dev stderr: ${stderr}`);
-    });
+    );
+
+    // Start the devtools server.
+    devtoolsProc = exec(
+      "npm exec next dev -- -p 8081",
+      {
+        env: {
+          ...process.env,
+          DASHBOARD_URL: "http://localhost:8080",
+        },
+      },
+      (error, stdout, stderr) => {
+        if (error) {
+          console.error(`devtools server ERROR: ${error}`);
+        }
+        console.error(`devtools server stdout: ${stdout}`);
+        console.error(`devtools server stderr: ${stderr}`);
+      }
+    );
 
     // Debug: Allow verifying the replay-cli version.
     console.log(`Checking @replayio/replay version...`);
     execSync("npx replay version", {
       stdio: "inherit",
     });
 
-    // make sure the dev server is up and running.
-    console.log("waiting for dev server to start up");
-    await testHttpConnection("http://localhost:8080/");
-    console.log("dev server up, continuing with test");
+    // make sure the servers are up and running.
+    console.log("waiting for dev servers to start up");
+    await testHttpConnection("http://localhost:8081/");
```

Comment:
I think we could check `http://localhost:8081/login` instead if we want to get a 200 without a possible redirect

Reply:
ah yeah I meant to add that :+1:

Reply:
I like this URL because it will not load at all if DevTools isn't running yet and it will return a 500 error if Dashboard isn't running. Otherwise it will always return a 200 status (regardless of authentication)